### ISSUE TITLE
Why the dimension of the patch logit for **each example**  is in  64*3 not only 3?

### ISSUE THREAD

**Issue by moz164164** [seasonSH/WarpGAN#5]
Why the dimension of the patch logit for **each example**  is in  64*3 not only 3?

related code shown below

def discriminator(images, num_classes, bottleneck_size=512, keep_prob=1.0, phase_train=True,
            weight_decay=0.0, reuse=None, scope='Discriminator'):
    with slim.arg_scope([slim.conv2d, slim.fully_connected],
                        weights_regularizer=slim.l2_regularizer(weight_decay),
                        activation_fn=leaky_relu,
                        normalizer_fn=None,
                        normalizer_params=batch_norm_params):
        with tf.variable_scope(scope, [images], reuse=reuse):
            with slim.arg_scope([slim.batch_norm, slim.dropout],
                                is_training=phase_train):

                print('{} input shape:'.format(scope), [dim.value for dim in images.shape])

                net =conv(images, 32, kernel_size=4, stride=2, scope='conv1')
                print('module_1 shape:', [dim.value for dim in net.shape])
                
                net = conv(net, 64, kernel_size=4, stride=2, scope='conv2')
                print('module_2 shape:', [dim.value for dim in net.shape])

                net = conv(net, 128, kernel_size=4, stride=2, scope='conv3')
                print('module_3 shape:', [dim.value for dim in net.shape])
 
             
                net = conv(net, 256, kernel_size=4, stride=2, scope='conv4')
                print('module_4 shape:', [dim.value for dim in net.shape])

                net = conv(net, 512, kernel_size=4, stride=2, scope='conv5')
                print('module_5 shape:', [dim.value for dim in net.shape])

                # Patch Discrminator
                patch5_logits = slim.conv2d(net, 3, 1, activation_fn=None, normalizer_fn=None, scope='patch5_logits')
                patch_logits = tf.reshape(patch5_logits, [-1,3])

**Comment by ronny3050** [comment]
`patch_logits = tf.reshape(patch5_logits, [-1,3])` reshapes patch logits to `batch_size * 3`.
